Patch by Robert Scheck <redhat@linuxnetz.de> for w3m >= 0.5.1, which makes
w3m rebuildable using gcc 4.

--- w3m-0.5.1/istream.c			2003-10-20 18:41:56.000000000 +0200
+++ w3m-0.5.1/istream.c.gcc4		2005-03-15 12:18:36.000000000 +0100
@@ -37,7 +37,7 @@
 {
     int len;
     base->stream.cur = base->stream.next = 0;
-    len = base->read(base->handle, base->stream.buf, base->stream.size);
+    len = (base->read) (base->handle, base->stream.buf, base->stream.size);
     if (len <= 0)
 	base->iseos = TRUE;
     else
@@ -320,7 +320,7 @@
     len = buffer_read(&base->stream, buf->ptr, count);
     rest = count - len;
     if (MUST_BE_UPDATED(base)) {
-	len = base->read(base->handle, &buf->ptr[len], rest);
+	len = (base->read) (base->handle, &buf->ptr[len], rest);
 	if (len <= 0) {
 	    base->iseos = TRUE;
 	    len = 0;
